src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -171,15 +175,32 @@ private void computeCodeActionsForSonarLintIssues(Diagnostic diagnostic, List<Ei
         newCodeAction.setCommand(new Command(fix.message(), SONARLINT_QUICK_FIX_APPLIED, List.of(ruleKey)));
         codeActions.add(Either.forRight(newCodeAction));
       });
+
+      if (!hasNoBinding) {
```

Comment:
this is quite confusing. Maybe have a variable `hasBinding = binding.isPresent()` and use that?